port_ocean/run.py

```diff
@@ -52,7 +54,7 @@ def _create_default_app(path: str | None = None) -> Ocean:
 
     spec = get_spec_file()
     config_factory = None
-    if spec is not None:
+    if spec is not None and use_default_config_factory:
```

Comment:
Why is that flag?
It should use default if there is no other implementation

Reply:
Yea it's a bit confusing, 
I don't want to validate configurations of the integration, I only want to validate port.clientId and port.clientSecret exists

Reply:
separate the default config factory to a separate function and geet the values as a parameter

`config_factory: Type[BaseModel] | None`